urbs/input.py

```diff
@@ -516,6 +516,15 @@ def pyomo_model_prep(data, timesteps):
     # dictionaries for additional features
     if m.mode['tra']:
         m.transmission_dict = transmission.to_dict()
+        # DCPF transmission lines are bidirectional and do not have symmetry
```

Comment:
Why are DCPF bidirectional? How is that understandable for users?
And why is it then necessary to multiply by 2?

Reply:
The equation of DC power flow includes the voltage angle difference between source and destination sites. Depending on the values of voltage angles, the power flow can have a negative value, indicating the direction of the power flow. For this reason, a transmission line modelled with DC power flow is introduced as bidirectional.
The users can still give two separate transmission arcs for a transmission line as in the previous version but if the transmission line has reactances defined, one of the complementary arcs will be removed from the model `remove_duplicate_transmission().` This way the users will not have to change any input method.
With the symmetry constraint for transmission lines, both of the transmission arcs of a transmission line must have the same total capacity installed. Which means, if a transmission capacity expansion takes place for an arc, it would cause an expansion in the complementary arc. Since the DCPF transmission lines do not have complementary arcs the fixed and investment costs would be halved. To prevent this error, fixed and investment prices for DCPF lines should be doubled before calculating the costs.

Reply:
> With the symmetry constraint for transmission lines, both of the transmission arcs of a transmission line must have the same total capacity installed. Which means, if a transmission capacity expansion takes place for an arc, it would cause an expansion in the complementary arc. Since the DCPF transmission lines do not have complementary arcs the fixed and investment costs would be halved. To prevent this error, fixed and investment prices for DCPF lines should be doubled before calculating the costs.

Alternatively, one might halve the costs for the non-DC transmission lines, if the input inv-cost reflects the actual cost of the transmission line (and not the rather unintuitive "half-cost" as it is now). We should discuss this at one point.